### DISCUSSION TITLE
Not Gate

### DISCUSSION THREAD

**Discussion by TannerJ07** [TodePond/Arroost#299] [Show and tell]
![Screenshot 2024-04-10 11 02 17 AM](https://github.com/TodePond/Arroost/assets/130108795/99e215f7-0ec1-4490-b162-2b1206c937da)
The top square is the input, the top circle sends out the output, and the bottom right circle is the output.

**Comment by TannerJ07** [comment]
I've done some more testing, and I honestly don't think it can be any simpler. This game would also function greatly with a copy paste mechanic.

**Comment by TodePond** [comment]
this is so cool!

**Comment by mahdoosh1** [comment]
> This game would also function greatly with a copy paste mechanic.

or if it had a way to compact everything down to one custom node

**Reply by TannerJ07** [reply]
Ooh cool idea, might get confusing with a lot of custom nodes, Depends on how it is implemented.
